styleguide/_themes/global/scss/vendor_prefixes.scss

```diff
@@ -41,6 +41,12 @@
   border-radius: $radius;
 }
 
+@mixin transform($degrees) {
```

Comment:
Maybe call this `rotate` instead of `transform` in case we ever want to make a mixin for other transform properties?